src/Package/Impl/Telemetry/RtvsTelemetry.cs

```diff
@@ -171,7 +172,11 @@ internal class WindowEvents {
                                 CompletionEnabled = _editorSettings.CompletionEnabled,
                                 SyntaxCheckInRepl = _editorSettings.SyntaxCheckInRepl,
                                 PartialArgumentNameMatch = _editorSettings.PartialArgumentNameMatch,
-                                RCommandLineArguments = _settings.LastActiveConnection?.RCommandLineArguments ?? string.Empty,
+
+                                // Do not report command line arguments - they may contain 
+                                // PII information and they are not that interesting anyway.
+                                // RCommandLineArguments = _settings.LastActiveConnection?.RCommandLineArguments ?? string.Empty,
```

Comment:
Shouldn't we just delete this line?

Reply:
I guess. Added comment so no one would put it back :-)